BaselineOfIceberg.package/BaselineOfIceberg.class/properties.json

```diff
@@ -1,11 +1,14 @@
 {
-	"commentStamp" : "",
-	"super" : "BaselineOf",
 	"category" : "BaselineOfIceberg",
-	"classinstvars" : [ ],
-	"pools" : [ ],
-	"classvars" : [ ],
-	"instvars" : [ ],
+	"classinstvars" : [
+		 ],
+	"classvars" : [
+		 ],
+	"commentStamp" : "",
+	"instvars" : [
+		 ],
 	"name" : "BaselineOfIceberg",
-	"type" : "normal"
-}
\ No newline at end of file
+	"pools" : [
+		 ],
+	"super" : "BaselineOf",
+	"type" : "normal" }
```

Comment:
The files are exported from FileTree right? So If I open an issue it is on FileTree?